app/models/student.rb

```diff
@@ -1,32 +1,26 @@
 class Student < User
 
   has_many :accepted_groupings,
+           -> { where 'memberships.membership_status' =>
+                        [StudentMembership::STATUSES[:accepted],
+                         StudentMembership::STATUSES[:inviter]] },
            class_name: 'Grouping',
            through: :memberships,
-           source: :grouping,
-           conditions: {
-             'memberships.membership_status' =>
-               [StudentMembership::STATUSES[:accepted],
-                StudentMembership::STATUSES[:inviter]]
-           }
+           source: :grouping
 
   has_many :pending_groupings,
+           -> { where 'memberships.membership_status' =>
+                        StudentMembership::STATUSES[:pending] },
            class_name: 'Grouping',
            through: :memberships,
-           source: :grouping,
-           conditions: {
-             'memberships.membership_status' =>
-               StudentMembership::STATUSES[:pending]
-           }
+           source: :grouping
 
   has_many :rejected_groupings,
+           -> { where 'memberships.membership_status' =>
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.